src/Sentry.Unity/UnityEventProcessor.cs

```diff
@@ -20,6 +21,13 @@ internal static class UnitySdkInfo
 
     internal class UnityEventProcessor : ISentryEventProcessor
     {
+        private readonly IApplication _appDomain;
+
+        public UnityEventProcessor(IApplication? appDomain = null)

```

Comment:
there's no AppDomain here:

```suggestion
        public UnityEventProcessor(IApplication? application = null)

```